app/login/page.test.tsx

```diff
@@ -76,4 +79,34 @@ describe('Login', () => {
 
     mockUseAuthContext.isSignedIn = false;
   });
-});
+
+  test('successfully logs in and shows success notification', async () => {
+    const pretendUser = {
+      email: 'testemail@email.com',
+      password: 'test1234',
+    }
+
+    await mockLoginAccount(pretendUser);
+    expect(account.createEmailPasswordSession).toBeInstanceOf(Object);
```

Comment:
There's no reason to test that this is an object. Also, createEmailPasswordSession is not a part of this file and doesn't need to be tested. Remove these test and test the alert logic in AuthContextProvider.